### PR TITLE
Add 'epsilon' and 'verbose' parameters to 'testing.diff'

### LINKED ISSUE #911: Document 'epsilon' parameter for 'testing.diff' function
The `epsilon` parameter defines the margin of error allowed when comparing floating point values.

##### Relevant URLs
- https://github.com/influxdata/flux/blob/master/stdlib/testing/diff.go#L20-L26

### PR DESCRIPTION
**Pull request by sanderson**
Closes #911 

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/v2.0/reference/flux/stdlib/testing/diff.md [hunk 1]
```diff
@@ -34,16 +39,27 @@ _The `testing.diff()` function can be used to perform in-line diffs in a query._
...
 
 _**Data type:** Object_
 
 ### want
-The stream that contains the expected data to test against.
+Stream that contains the expected data to test against.
 
 _**Data type:** Object_
 
+### verbose
```
Comment (kelseiv) [inline content/v2.0/reference/flux/stdlib/testing/diff.md:52]: default true?

Comment (sanderson) [reply content/v2.0/reference/flux/stdlib/testing/diff.md:52]: I'm not sure 😄... @yzhang1991 ?

Comment (yzhang1991) [reply content/v2.0/reference/flux/stdlib/testing/diff.md:52]: Actually I don't know about `verbose`
From the code, it does not seem to be used anywhere. I think it is dead code.

Comment (sanderson) [reply content/v2.0/reference/flux/stdlib/testing/diff.md:52]: Nm, default is `true`. Added it to the description.

Comment (yzhang1991) [reply content/v2.0/reference/flux/stdlib/testing/diff.md:52]: Default is true, but we did not use it in the Process() code @nathanielc

Comment (sanderson) [reply content/v2.0/reference/flux/stdlib/testing/diff.md:52]: @yzhang1991 if it's not used, should I remove it or leave it there?

Comment (yzhang1991) [reply content/v2.0/reference/flux/stdlib/testing/diff.md:52]: @sanderson I suggest you remove it. Setting `verbose` does not change the function behavior in any way.

File: content/v2.0/reference/flux/stdlib/testing/diff.md [hunk 2]
```diff
@@ -34,16 +38,22 @@ _The `testing.diff()` function can be used to perform in-line diffs in a query._
...
 
 ### want
-The stream that contains the expected data to test against.
+Stream that contains the expected data to test against.
 
 _**Data type:** Object_
 
+### epsilon
+Specifies how far apart two **float** values can be, but still considered equal.
+Defaults to `0.000000001`.
```
Comment (yzhang1991) [inline content/v2.0/reference/flux/stdlib/testing/diff.md:53]: I would recommend this to become something like `1e-9`, hard to count those zeros.

Comment (sanderson) [reply content/v2.0/reference/flux/stdlib/testing/diff.md:53]: I agree, but Flux doesn't support float values using scientific notation and I don't want to confuse users.

Comment (yzhang1991) [reply content/v2.0/reference/flux/stdlib/testing/diff.md:53]: I see, thanks!

Comment (yzhang1991) [reply content/v2.0/reference/flux/stdlib/testing/diff.md:53]: I created https://github.com/influxdata/flux/issues/2732, hoping that we could have scientific notation supported one day.

**Pull request merged**